docker-compose.yml

```diff
@@ -22,23 +22,25 @@ services:
     environment:
       - PGADMIN_DEFAULT_EMAIL=assistant@example.com
       - PGADMIN_DEFAULT_PASSWORD=password
-      
+
   flyway:
     image: flyway/flyway:7.15.0
     depends_on:
       - db
     volumes:
       - ./sql:/flyway/sql
     command: -url=jdbc:postgresql://db/mydb -user=postgres -password=password migrate
-  
-  app:
-    build:
-      context: .
-      dockerfile: Dockerfile
-    depends_on:
-      - db
-      - flyway
+  mongo:
+    container_name: mongo_db
+    image: mongo
+    restart: always
+    volumes:
+      - mongo_data:/data/db
     ports:
-      - 8888:8888
+      - 27017:27017
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: VirtualAssistant60089
```

Comment:
dont expose user name and passwords use here: ` MONGO_INITDB_ROOT_USERNAME: System.getenv("MONGO_USERNAME")
                MONGO_INITDB_ROOT_PASSWORD: System.getenv("MONGO_PASSWORD")` so it will be able to read the properties from system environment